core/tests/puppeteer-acceptance-tests/blog-admin-tests/blog-admin-assign-roles-to-users-and-change-tag-properties.js

```diff
@@ -0,0 +1,75 @@
+// Copyright 2023 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Accpetance Test for a Blog Admin to assign roles and change
+ * tag properties from the blog-admin page.
+ */
+
+const e2eSuperAdmin = require(
+  '../puppeteer-testing-utilities/blogPostAdminUtils.js');
+const e2eBlogAdmin = require(
+  '../puppeteer-testing-utilities/blogPostAdminUtils.js');
+const e2eBlogPostEditor = require(
+  '../puppeteer-testing-utilities/blogPostAdminUtils.js');
+const testConstants = require(
+  '../puppeteer-testing-utilities/testConstants.js');
+
+const blogAdminUrl = testConstants.URLs.BlogAdmin;
+const ROLE_BLOG_ADMIN = 'BLOG_ADMIN';
+const ROLE_BLOG_POST_EDITOR = 'BLOG_POST_EDITOR';
+
+let blogAdminAssignRolesToUsersAndChangeTagProperties = async function() {
+  const superAdmin = await new e2eSuperAdmin();
+  const blogAdmin = await new e2eBlogAdmin();
+  const blogPostEditor = await new e2eBlogPostEditor();
+
+  await blogAdmin.openBrowser();
+  await blogAdmin.signUpNewUserWithUsernameAndEmail(
```

Comment:
Why not just `signUpNewUser`, will there be different signup possibilities?

Reply:
changed to `signUpNewUser` at all places.